### DISCUSSION TITLE
Recording the questions and it's answers

### DISCUSSION THREAD

**Discussion by 53mops** [voidbydefault/EasyApplyBot#42] [Ideas]
It would be great if the bot can also record the questions and the answers it gives to those questions while applying to the jobs in output csv (or another csv), because sometimes I catch it giving incorrect answers. 

It would be great to track what edge cases is throwing it off.

For example I would be able to see what skills the jobs posters are requesting skills/years of experience in and modify config file for that.

Similarly I would have a lot more data to optimise the bot if it recorded all it's questions and answers.

**Comment by voidbydefault** [comment]
Hi @53mops ,

Apologies for not being able to answer early. It's possible, let me see it later. I am going through some personal matters, it may take a while to implement. Thanks.
